compute_endpoint/globus_compute_endpoint/endpoint/config/example_identity_mapping_config.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "comment": "For more examples, see: https://docs.globus.org/globus-connect-server/v5.4/identity-mapping-guide/",
+    "DATA_TYPE": "external_identity_mapping#1.0.0",
+    "command": ["/bin/false", "--some", "flag", "-a", "-b", "-c"]
+  },
+  {
+    "comment": "For more examples, see: https://docs.globus.org/globus-connect-server/v5.4/identity-mapping-guide/",
+    "DATA_TYPE": "expression_identity_mapping#1.0.0",
+    "mappings": [
+      {
+        "source": "{username}",
+        "match": "(.*)@example.com",
```

Comment:
We should make it clear in our docs that a MU endpoint will not work until this is updated.

Reply:
FWIW, it actually _would_ work, provided a username that has an `@example.com` suffix.

Reply:
(But I take your larger point as "we need documentation in general."  And yes, that's coming, but not in this PR.)

Reply:
Lol, right. I originally thought we should attempt to make it work with a more general default expression to match the first identity in the set, but it's probably better to have certain failure that is well-documented.